galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -222,7 +222,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
             BundleManagement.loadBundle(repositoryAdmin, bundleContext, testBundleName);
         } catch (Exception e) {
             logger.error("Unable to load the test bundle " + testBundleName, e);
-            updateStatus("finished", "finished");
+            updateStatus(TestRunLifecycleStatus.FINISHED, "finished");
```

Comment:
These two lines are repeated. violating the minimal elements/"Don't repeat yourself" rules... 
Perhaps adding a new method testFinished() which does both ?